roles/pulibrary.patroni/tasks/manage_users.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: create /update pg_hba.conf
```

Comment:
This is `default.yml` in my Vagrant Box, but is successfully created with the desired values.